cmd/local-gadget/main.go

```diff
@@ -407,19 +424,81 @@ func main() {
 				log.StandardLogger().SetLevel(log.DebugLevel)
 			}
 
+			parts := strings.Split(runtimes, ",")
+			if len(parts) > len(containerutils.AvailableRuntimes) {
+				return utils.WrapInErrInvalidArg("--runtime / -r",
+					errors.New("maximum values for this flag was exceeded"))
+			}
```

Comment:
Mmmm this error is actually to tell users that it is not possible to use more than N container runtimes. It is trying to avoid situations like users specifying `-r docker,cri-o,docker` and we wrongly adding twice the `containercollection.WithContainerRuntimeEnrichment()` with docker.

If the runtime is not supported, the error will be indeed different: https://github.com/kinvolk/inspektor-gadget/blob/fb3fa3f6c9492beb742ee64c6e411c6dc1339236/cmd/local-gadget/main.go#L435-L438. 

Anyway, I modified the logic to avoid it being misleading.